dll/win32/syssetup/syssetup.spec

```diff
@@ -53,7 +53,7 @@
 @ stub SetupGetProductType
 @ stub SetupGetSetupInfo
 @ stub SetupGetValidEula
-@ stub SetupInfObjectInstallActionW
+@ stdcall SetupInfObjectInstallActionW(long long wstr long) SETUPAPI.InstallHinfSectionW
```

Comment:
As you can see e.g. here: https://git.reactos.org/?p=reactos.git&a=search&h=HEAD&st=grep&s=kernel32. or there: https://git.reactos.org/?p=reactos.git&a=search&h=HEAD&st=grep&s=ntdll. , redirections do not need to have the list of parameters to be specified.

Reply:
It is better to leave it off even, since a forwarder is no more than a string in the format of `dllname.functionname`